content/en/boilerplates/revision-tags-middle.md

```diff
@@ -12,11 +12,11 @@ The cluster operator can view this mapping in addition to tagged namespaces thro
 {{< text bash >}}
 $ istioctl tag list
 TAG         REVISION NAMESPACES
-default     1-9-5    ...
-prod-canary 1-10-0   ...
-prod-stable 1-9-5    ...
+default     {{< istio_previous_version_revision >}}-5   ...
```

Comment:
There often is no `1-x-5` version of a previous release when a new one comes out. The only guarantee is that there is a `1-x-0`, although it's probably safe to assume there was at least one patch release, so I would use 0 or 1 for the previous revision:

```suggestion
default     {{< istio_previous_version_revision >}}-1   ...
```